db_upgrade_scripts/mosip_idrepo/sql/1.2.0.1_to_1.2.1.0_upgrade.sql

```diff
@@ -0,0 +1 @@
+\echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
```

Comment:
identity_update_count_tracker  table is added. Please add the DB scripts for that.

Reply:
https://github.com/mosip/id-repository/blob/0d56e166ff64713732a493262b63c5bf409cfcca/db_upgrade_scripts/mosip_idrepo/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql#L104

it's already there in earlier upgrade script i.e. 1155 to 1201b1